src/plugins/iconv/iconv.c

```diff
@@ -26,6 +26,11 @@
 
 
 #include "iconv.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <errno.h>
+
+#define CHAR_SIZE_MAX 4
```

Comment:
The older standard RFC 2279 seems to also know 6 bytes. But thats not the point, it seems to be very inefficient to handle every character seperately. Any specific reason why kdbbUTF8Engine is not used?

If the kdbbUTF8Engine implementation is not good, it should be fixed.